pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -220,9 +221,10 @@ stdenv.mkDerivation (finalAttrs: {
     "-Dsysconfdir_install=${placeholder "out"}/etc"
     "-Defi_os_dir=nixos"
     "-Dplugin_modem_manager=enabled"
-
     # We do not want to place the daemon into lib (cyclic reference)
     "--libexecdir=${placeholder "out"}/libexec"
+  ] ++ lib.optionals (!enablePassim) [
+    "-Dpassim=disabled"
```

Comment:
Unless this service opens a tcp port default, should we not just enable it?

Reply:
Yeah, I will look into it, didn't have the time yet. We can do this in a separate PR as well, since we'll need to package passim first.

Passim seems to be a separate systemd service, so we can compile fwupd with passim support enabled, and then add a nixos module option to optionally enable the systemd service.